apps/web/src/components/layout/AppLayout.tsx

```diff
@@ -60,17 +60,9 @@ export function AppLayout() {
                 </SpotLight>
               </Sentry.ErrorBoundary>
             </AppShell>
-          </SupportChatProvider>
+          </IntercomProvider>
         </ThemeProvider>
       </SpotLightProvider>
     </RequiredAuth>
   );
 }
-
-function SupportChatProvider({ children }) {
-  if (INTERCOM_APP_ID) {
-    return <IntercomProvider appId={INTERCOM_APP_ID}>{children}</IntercomProvider>;
-  }
-
-  return children;
-}
```

Comment:
relates to conditionally calling hook, the code change in a file below...
when there is no `INTERCOM_APP_ID` the library doesn't show any warnings until it's initialized... but we don't do that if we don't have the `INTERCOM_APP_ID` value